### PR TITLE
Exercise ppj 05 III

### PR DESCRIPTION
**Pull request by lukaszkutylowski**

### CODE REVIEW

File: Fibonacci.java [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-	int intUserInput = input.nextInt();
-	int serieArrayNumber = intUserInput - 1;
-	
-	System.out.print("value of " + intUserInput + " fibonacci serie: ");
-	System.out.println(fibonacciSeries[serieArrayNumber]);
-	
-	System.out.print("value of " + intUserInput + " fibonacci series: ");
-	System.out.println(fibonacciSeries[intUserInput]);
-	}
-}
```
Comment (MatDls) [inline Fibonacci.java]: Hey @lukaszkutylowski 
Do you understand what's going on here?
Please, explain if this was intentional.
If not, then please correct.
If yes - than please explain it to me.
Use @ sign to mention me somewhere or just call If that's easier

**Review by MatDls — CHANGES_REQUESTED** [review]

File: Fibonacci.java (hunk 1 above)
Comment (MatDls) [reply Fibonacci.java]: Hey Man!
Why you didn't answer?
:)

Don't be shy, reply:)

**Comment by lukaszkutylowski** [comment]
@MatDls 
so it's all good? I can merge?

**Comment by lukaszkutylowski** [comment]
@MatDls 
maybe I wrote not clearly, but position of array is number from 0, but Fibonacci series have not 0 position - 1st element of series, 2nd element etc. but 0st element? rather not. zero position in array is first position of series. in this situation if user want see 1st element program must have 0 position of array.

**Pull request merged**